flow/raster_cache_key.h

```diff
@@ -13,30 +13,30 @@
 
 namespace flutter {
 
-enum class RasterCacheKeyType { kLayer, kPicture, kDisplayList };
+enum class RasterCacheKeyKind { kLayer, kPicture, kDisplayList };
```

Comment:
I think my previous comment may have been misunderstood. My suggestion was not to replace "type" with "kind", but to have "type" stay the way it was, which was `kLayer, kPicture, kDisplayList` and then also have a new classification, such as "kind" or "metric_type" which specified which metric it contributed to, as in `kLayer, kPicture` or `kLayerMetrics, kPictureMetrics`.

The kind/metrics_type could be inferred from the regular type. The inference would only have to be done in either the constructor or a getter method on the RasterCacheKey and then the code that dealt with the metrics would only have to deal with the 2 "kind/metrics_type"s rather than the 3 "type"s.

But, with the new code using a switch statement, the tests for "type" are less intrusive anyway. However there are still places where the code has to do `type == kPicture || type == kDisplayList` when they could do `mode/metrics_type == kPictureMetrics` (same for kLayerMetrics).

Reply:
Ahhh, I see. Thanks for your comment, I changed the code according to it :-) .